CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ FetchContent_MakeAvailable(dlpack)
 #
 ExternalProject_Add(
   boostorg
-  URL https://boostorg.jfrog.io/artifactory/main/release/1.79.0/source/boost_1_79_0.tar.gz
+  URL https://archives.boost.io/release/1.79.0/source/boost_1_79_0.tar.gz
```

Comment:
Can we align versions with installed one in container which 1.80.0 ?

Reply:
If it's OK with you, I'll leave this one as-is to see if it fixes CI. 

Then I'll make a separate PR to lift version to 1.80 to match. I don't know if we could be doing something smarter to re-use assets from server build rather than re-download here though.